include/glow/Runtime/RuntimeTypes.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_RUNTIME_RUNTIMETYPES_H
+#define GLOW_RUNTIME_RUNTIMETYPES_H
+
+#include "glow/Backends/BackendUtils.h"
+#include "glow/Graph/Graph.h"
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace glow {
+namespace runtime {
+
+using NetworkIDty = size_t;
+using DeviceIDty = size_t;
+using RunIdentifierTy = size_t;
+
+/// Enum to communicate results when communicating with device at initialization
+/// and runtime.
+enum ResultCode { READY, EXECUTED, FAILED, CANCELLED };
```

Comment:
Can you make this an `enum class`, and make them lower case for consistency with the rest of the code base.

Reply:
Ah yes that got lost when i pulled in RuntimeTypes, what do you mean by the second part? Is there a good example of the style you'd like me to match?

Reply:
All of our other `enum classes` have UpperCamelCase values, e.g. `CompilationMode` has `Train` and `Infer` (rather than `TRAIN` and `INFER`).